src/ferui-components/tree-view/index.ts

```diff
@@ -0,0 +1 @@
+export * from './interfaces';
```

Comment:
You also need to add your components there too. It will be used to generate the public-api.

Reply:
Added